### DISCUSSION TITLE
New icons are hard to identify; though malware was installed accidentally

### DISCUSSION THREAD

**Discussion by Darthagnon** [darkreader/darkreader#7572] [Ideas]
I was disappointed to see the familiar Darkreader icons ditched in [v4.9.43](https://github.com/darkreader/darkreader/releases/tag/v4.9.43). For a minute, I thought perhaps I'd made a mistake and installed malware (and I'm knowledgeable with PCs). The pastel colours and excessive roundness make them fade and disappear among the dozens of other extension icons I have. I don't like them and will be reverting to [v4.9.42](https://github.com/darkreader/darkreader/releases/tag/v4.9.42) for the foreseeable future; I thought I'd probably have to revert once v5 released, but I guess the "golden version" was a little bit earlier. 

Why the need for a brand refresh? House of Mouse annoyed at Not-Vader and Not-Luke?

**Comment by Darthagnon** [comment]
I tried to install v4.9.42 from GitHub releases, but my Ungoogled Chromium complained about "CRX Header Invalid". Does anyone have an archived CRX from the Chrome Web Store? EDIT: I managed to get a working CRX from [here](https://www.crx4chrome.com/crx/1057/). Strange that working CRX files are not included in the GitHub Releases.

**Reply by Gusted** [reply]
> Strange that working CRX files are not included in the GitHub Releases.

They are included, but they aren't in a CRX file, as that's almost the same as how zip works, which is why it's published in zip(they need to be unpacked, before chrome can use it).


The icons were changed to more look like a mascot/human. But I can see why it might be confusing for someone.

CC: @alexanderby could you enlighten this a bit?

**Reply by Darthagnon** [reply]
I know about renaming ZIPs to CRX, and I did so with the appropriate release zips, but they didn't work; that's what I meant by missing "working" files. Probably some manifest file thing that gets added to store releases.

**Reply by Gusted** [reply]
You can't convert zip to CRX(other way around you can). None the less, you should unpack the zip in order for it to become nicely for chrome://extension

**Comment by alexanderby** [comment]
Hi. The colors remained the same, only the goggles were enlarged and the head shape changed a bit. Just in case. We live in a world where [a company can sue for a similar pear leaf](https://www.creativebloq.com/news/apple-prepear-logo-resolution).
